bait-files/src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps;
```

Comment:
This file looks out of date? Fix this. Don't have in-progress stuff assigned to us for review :)

Reply:
Added progress